lib/app_web/controllers/item_controller.ex

```diff
@@ -0,0 +1,37 @@
+defmodule AppWeb.ItemController do
```

Comment:
@SimonLab why are we writing `Controllers` in a `LiveView` App? 🤷‍♂️ 
(apologies if I'm missing something ... is this how it's done? 💭)

Reply:
This controller contains only the `edit` and `udpate` actions:
https://github.com/dwyl/mvp/blob/68f556225d568e2e3e90ff8092f8041f8fb0b6c5/lib/app_web/router.ex#L39

It is used for updating which lists an item can be added to